.github/workflows/check-branch-commits.yaml

```diff
@@ -5,7 +5,7 @@ name: "check-branch-commits"
 on:
   workflow_dispatch:
   schedule:
-    - cron: "0 0,12 * * *" # At hour 0 and 12 - AKA twice a day (UTC)
+    - cron: "0 0,12 * * *" # At hour 0 and 12 - AKa twice a day (UTC)
```

Comment:
sup?

Reply:
Aah, this was me touching the file so the job would run on this PR 😄